src/qt/votingdialog.cpp

```diff
@@ -762,15 +822,297 @@ VotingVoteDialog::VotingVoteDialog(QWidget *parent)
+    question->setTextInteractionFlags(Qt::TextSelectableByMouse);
+    glayout->addWidget(question, 2, 0);
+
+    question_ = new QLineEdit();
+    question_->setAlignment(Qt::AlignLeft|Qt::AlignVCenter);
+    glayout->addWidget(question_, 2, 1);
+
+    //url
+    QLabel *discussionLabel = new QLabel(tr("Discussion URL: "));
+    discussionLabel->setAlignment(Qt::AlignLeft|Qt::AlignVCenter);
+    discussionLabel->setTextInteractionFlags(Qt::TextSelectableByMouse);
+    glayout->addWidget(discussionLabel, 3, 0);
+
+    url_ = new QLineEdit();
+    url_->setAlignment(Qt::AlignLeft|Qt::AlignVCenter);
+    glayout->addWidget(url_, 3, 1);
+    
+    //share type
+    QLabel *shareType = new QLabel(tr("Share Type: "));
+    shareType->setAlignment(Qt::AlignLeft|Qt::AlignVCenter);
+    shareType->setTextInteractionFlags(Qt::TextSelectableByMouse);
+    glayout->addWidget(shareType, 4, 0);    
+    
+    shareTypeBox_ = new QComboBox(this);
+    QStringList shareTypeBoxItems;
+    shareTypeBoxItems << "Magnitude" << "Balance" << "Both" << "CPIDCount" << "ParticipantCount";
+    shareTypeBox_->addItems(shareTypeBoxItems);
+    shareTypeBox_->setCurrentIndex(2);
+    glayout->addWidget(shareTypeBox_, 4, 1);
+    
+    //answers
+    answerList_ = new QListWidget(this);
+    answerList_->setContextMenuPolicy(Qt::CustomContextMenu);
+    connect(answerList_, SIGNAL(customContextMenuRequested(const QPoint &)), this, SLOT(showContextMenu(const QPoint &)));
+    vlayout->addWidget(answerList_);
+    connect (answerList_, SIGNAL (itemDoubleClicked (QListWidgetItem *)), this, SLOT (editItem (QListWidgetItem *)));
+    
+    QHBoxLayout *hlayoutTools = new QHBoxLayout();
+    vlayout->addLayout(hlayoutTools);
+    
+    QPushButton *addItemButton = new QPushButton();
+    addItemButton->setText(tr("Add Item"));
+    hlayoutTools->addWidget(addItemButton);
+    connect(addItemButton, SIGNAL(clicked()), this, SLOT(addItem()));
+
+    QPushButton *removeItemButton = new QPushButton();
+    removeItemButton->setText(tr("Remove Item"));
+    hlayoutTools->addWidget(removeItemButton);
+    connect(removeItemButton, SIGNAL(clicked()), this, SLOT(removeItem()));
+
+    QPushButton *clearAllButton = new QPushButton();
+    clearAllButton->setText(tr("Clear All"));
+    hlayoutTools->addWidget(clearAllButton);
+    connect(clearAllButton, SIGNAL(clicked()), this, SLOT(resetData()));
+    
+    QHBoxLayout *hlayoutBottom = new QHBoxLayout();
+    vlayout->addLayout(hlayoutBottom);
+
+    QPushButton *pollButton = new QPushButton();
+    pollButton->setText(tr("Create Poll"));
+    hlayoutBottom->addWidget(pollButton);
+    connect(pollButton, SIGNAL(clicked()), this, SLOT(createPoll()));
+    
+    pollNote_ = new QLabel();
+    pollNote_->setAlignment(Qt::AlignLeft|Qt::AlignVCenter);
+    pollNote_->setTextInteractionFlags(Qt::TextSelectableByMouse);
+    pollNote_->setWordWrap(true);
+    hlayoutBottom->addWidget(pollNote_);
+    
+    QString sPollTitle;
```

Comment:
remove what exactly? The pollNote_ label shows the response from the voting mechanism. Right now the only feedback if the poll creation was successfull